pylsp/_utils.py

```diff
@@ -14,6 +14,7 @@
 import jedi
 
 JEDI_VERSION = jedi.__version__
+CALL_TIMEOUT = 10
```

Comment:
Perhaps this constant should have a more specific name? I mean, `CALL_TIMEOUT` sounds too generic to me. What about `PROGRESS_CALL_TIMEOUT`?

Reply:
I removed the need for it because I removed the usage of this timeout for progress. 

It's back having the same name as before in `tests.test_language_server`